app/src/main/java/com/swyp/firsttodo/presentation/habit/list/HabitListViewModel.kt

```
@@ -0,0 +1,98 @@
+                }
+
+                updateState { copy(habits = Async.Success(newHabits)) }
+            }
+        }
+
+        fun onCreateClick() {
+            sendEffect(HabitListSideEffect.NavigateToHabitDetail(null))
+        }
+
+        fun onCheckClick(habit: Habit) {
+        }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)